tomviz/pvextensions/CMakeLists.txt

```diff
@@ -20,8 +20,17 @@ add_paraview_plugin(tomvizExtensions "1.0"
 )
 target_link_libraries(tomvizExtensions PUBLIC pqApplicationComponents)
 
-install(TARGETS tomvizExtensions
-        RUNTIME DESTINATION "bin"
-        LIBRARY DESTINATION "lib"
-        ARCHIVE DESTINATION "lib"
-	COMPONENT "runtime")
+if (NOT APPLE)
+  install(TARGETS tomvizExtensions
+          RUNTIME DESTINATION "bin"
+          LIBRARY DESTINATION "lib"
+          ARCHIVE DESTINATION "lib"
+          COMPONENT "runtime")
+else()
+  install(TARGETS tomvizExtensions
+          RUNTIME DESTINATION "bin"
+          LIBRARY DESTINATION "Applications/tomviz.app/Contents/Libraries"
```

Comment:
This didn't change.  The install tree was using @rpath in the install name of the tomvizExtensions  library and the fixup_bundle script doesn't handle that very well.  So rather than debug the script for hours, I changed the install code so that the fixup_bundle script doesn't need to find this file since it is already installed correctly.

I didn't know you could do that BUNDLE part of the install command on non-executables...  I'll try it and see if it works.